pipelines/build/common/build_base_file.groovy

```diff
@@ -127,7 +127,13 @@ class Builder implements Serializable {
         if (additionalBuildArgs) {
             buildArgs += ' ' + additionalBuildArgs
         }
-        def enableReproducibleCompare = isEnableReproducibleCompare(platformConfig, variant)
+
+        def configEnableReproducibleCompare = false
+        if (enableReproducibleCompare) {
+            // Pipeline parameter "enableReproducibleCompare" requests reproducibleCompare if enabled for this platform
+            configEnableReproducibleCompare = isEnableReproducibleCompare(platformConfig, variant)
+        }
+
```

Comment:
I think no need to add a new variable. No need to define a local one either.  The following should work

```
if (enableReproducibleCompare) {
            // Pipeline parameter "enableReproducibleCompare" requests reproducibleCompare if enabled for this platform
            enableReproducibleCompare = isEnableReproducibleCompare(platformConfig, variant)
        }
```

Reply:
yep looks good